### PR TITLE
Expose core action functions 

### PR DESCRIPTION
**Pull request by parro-it**
As discussed in #30

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -50,14 +50,62 @@ app.on('ready', () => {
...
 ```
 
-### Options
+## API
 
-#### showDevTools
+### Default exported function
+
+```js
+electronDebug({ showDevTools = false });
```
Comment (sindresorhus) [inline readme.md]: no point in showing defaults here. that just looks confusing. same with the other places.

File: readme.md [hunk 2]
```diff
@@ -50,14 +50,62 @@ app.on('ready', () => {
...
+```
+
+Install keyboard shortcuts and optionally
+activate DevTools on each created `BrowserWindow`.
```
Comment (sindresorhus) [inline readme.md]: don't hardwrap

File: readme.md [hunk 3]
```diff
@@ -50,14 +50,62 @@ app.on('ready', () => {
...
+```
+
+Toggle DevTools for specified `BrowserWindow` instance or for focused one if not specified.
+
+#### Options
+
+* win
+
+Type: `BrowserWindow`  
+Default: `undefined`
```
Comment (sindresorhus) [inline readme.md]: This is not correct as it's will use the focused one. Same with the below.

Comment (parro-it) [inline readme.md]: Do you refer to "Show DevTools on each created `BrowserWindow`.". 
I think this is the behaviour actived by the option: each BW created will trigger devtools open...

Comment (sindresorhus) [inline readme.md]: No, you have `Default: undefined`, while right above you say `or for focused one if not specified.`, so the default is not `undefined`, but rather the focused window.

Comment (parro-it) [inline readme.md]: Ah yes, now I got you! You're right, I'll change that.

**Pull request closed without merging**